installing/installing-preparing.adoc

```diff
@@ -234,7 +234,7 @@ endif::openshift-origin[]
 .User-provisioned infrastructure options
 |===
 ifndef::openshift-origin[]
-||AWS |Azure |Azure Stack Hub |GCP |{rh-openstack} |{rh-openstack} on SR-IOV |RHV |Bare metal |vSphere |VMC |IBM Z |IBM Z with {op-system-base} KVM |IBM Power |Platform agnostic
+||AWS |Azure |Azure Stack Hub |GCP |{rh-openstack} |{rh-openstack} on SR-IOV |RHV |Bare metal (x86_64) |Bare metal (ARM) |vSphere |VMC |IBM Z |IBM Z with {op-system-base} KVM |IBM Power |Platform agnostic
```

Comment:
When we say x86_64, then arm64  (or) aarch64 looks better.

Changes looks like:
Bare metal (x86_64) |Bare metal **(arm64)**

Reply:
Thats a good point! I remember I only used that interface because thats how it looked in the infrastructure provider. I'll change that up!